src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -304,32 +304,40 @@ <h3 id="history">
                   id="force_enable_addon" type="button">{{ _('Force-enable add-on') }}</button>
         </li>
         <li {% if addon.auto_approval_disabled %}class="hidden"{% endif %}>
-          <button data-api-url="{{ url('reviewers-addon-disable-auto-approval', addon.pk) }}"
+          <button data-api-url="{{ url('reviewers-addon-flags', addon.pk) }}"
+                  data-api-method="patch"
+                  data-api-data="{&quot;auto_approval_disabled&quot;: true}"
```

Comment:
Hmm.  Is using `&quot;` better than `data-api-data='"auto_approval_disabled": true}'`?

Reply:
I haven't tried it, I guess both works. I didn't want to use different quote characters than the rest of the HTML, do you have a strong opinion one way or the other?

Reply:
no strong opinions on this

Reply:
I'll merge it as-is then as I've spent already way too much time on this refactor and don't want to retest everything...